cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1246,6 +1246,14 @@
     </description>
   </property>
 
+  <!-- Market Configuration -->
+  <property>
+    <name>market.base.url</name>
+    <value>http://market.cask.co/v2</value>
+    <description>
+      The Base URL of the Cask Market RESTful API.
```

Comment:
Is this accurate as a description? Is this property not:
```
The base URL of the Cask Market used by the CDAP UI for the Cask Market RESTful API. 
The default value shown is that of the public Cask Market.
```
A reference to this property should be added to http://docs.cask.co/cdap/4.1.0-SNAPSHOT/en/integrations/cask-market.html#hosting-a-custom-cask-market